Dockerfile

```diff
@@ -35,6 +35,7 @@ RUN addgroup -g 1000 -S appgroup && \
 RUN mkdir -p /usr/src/app && mkdir -p /usr/src/app/tmp
 WORKDIR /usr/src/app
 
+COPY Gemfile* .ruby-version ./
 COPY Gemfile* ./
```

Comment:
Sorry I missed this.
Now there is a duplication. This has to be removed although does not break the image.